JapaneseFilter/sections/specific.txt

```diff
@@ -198,6 +198,7 @@ magazine.cainz.com###recommendedPost div.recommended:has(> div.related_content i
 pref.ehime.jp##.top_banner_7
 search.yahoo.co.jp#%#//scriptlet('json-prune', 'props.initialProps.pageProps.pageData.brandingAds')
 search.yahoo.co.jp##.AdBannerCarousel
+asobicreate.net#%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')
```

Comment:
Please move it to `antiadblock.txt` - https://github.com/AdguardTeam/AdguardFilters/blob/master/JapaneseFilter/sections/antiadblock.txt#L12-L13
It should be added with a comment to issue, and it would be nice to add a rule for platforms which do not support scriptlets, for example:
```adblock
asobicreate.net##div[id][class^="popup"][class$="wrap"][style^="opacity"]
```

So it should be like:
```
! https://github.com/AdguardTeam/AdguardFilters/issues/177410
asobicreate.net#%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')
asobicreate.net##div[id][class^="popup"][class$="wrap"][style^="opacity"]
```

Reply:
@AdamWr you don't need the prevent-fetch. the cosmetic rule is sufficient.

Reply:
Thank you, I as I am still learning... Can you look at #176305 later?